### PR TITLE
remove duplicate references to crashMenu outside scope

### PR DESCRIPTION
**Pull request by AndrewMeadows**
Somehow bad code was merged into v0.84.0-rc branch, breaking all subsequent PRs trying to get into that branch.  This PR fixes the problem.

### CODE REVIEW

File: interface/src/Menu.cpp [hunk 1]
```diff
@@ -769,44 +769,6 @@ Menu::Menu() {
 
         addActionToQMenuAndActionHash(crashMenu, MenuOption::CrashOnShutdown, 0, qApp, SLOT(crashOnShutdown()));
     }
-
```
Comment (AndrewMeadows) [inline interface/src/Menu.cpp]: These lines were duped and put into an `if (QProcessEnvironment::systemEnvironment().contains(HIFI_SHOW_DEVELOPER_CRASH_MENU)) {` context above, but were not removed, but the `crashMenu` instance only has scope inside the `if` block --> code would not compile.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21310/HighFidelity-Beta-PR16193-177fe5c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21310/HighFidelity-Quest-Beta-PR16193-177fe5c.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21310/HighFidelity-Beta-PR16193-177fe5c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21310/HighFidelity-Beta-Interface-PR16193-177fe5c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21310/HighFidelity-Beta-PR16193-177fe5c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21310/HighFidelity-Beta-Interface-PR16193-177fe5c.exe

**Pull request merged**